src/app/content/usage/usage-list/usage-list.component.html

```diff
@@ -6,31 +6,46 @@
                     <hc-label>Filter:</hc-label>
                     <input hcInput (keyup)="applyFilter()" [formControl]="searchControl" placeholder="Search" />
                 </hc-form-field>
-                <hc-form-field class="usage-table-filters-filter" inline>
-                    <hc-label>Category:</hc-label>
-                    <hc-select [formControl]="selectedCategoriesControl" (change)="applyFilter()">
-                        <option *ngFor="let cat of categories" [ngValue]="cat">{{ cat }}</option>
-                    </hc-select>
-                </hc-form-field>
+                <div class="usage-table-filters_categories">
+                    <hc-form-field class="usage-table-filters-filter" inline>
+                        <hc-label>Term Type:</hc-label>
+                        <hc-select [formControl]="selectedTypesControl" (change)="applyFilter()">
+                            <option *ngFor="let type of types" [ngValue]="type">{{ type }}</option>
+                        </hc-select>
+                    </hc-form-field>
+                    <hc-form-field class="usage-table-filters-filter" inline>
+                        <hc-label>Category:</hc-label>
+                        <hc-select [formControl]="selectedCategoriesControl" (change)="applyFilter()">
+                            <option *ngFor="let cat of categories" [ngValue]="cat">{{ cat }}</option>
+                        </hc-select>
+                    </hc-form-field>
+                </div>
+
             </div>
 
-            <table hc-table *ngIf="termList$ | async; else emptyList" [dataSource]="dataSource" style="border: none;">
+            <table hc-table *ngIf="termList$ | async; else emptyList" [dataSource]="dataSource">
                 <!-- Term Column -->
                 <ng-container hcColumnDef="term">
-                    <th hc-header-cell *hcHeaderCellDef>Term</th>
-                    <td hc-cell *hcCellDef="let usageItem">{{ usageItem.TermName }}</td>
+                    <th hc-header-cell style="width: 15%;" *hcHeaderCellDef>Term</th>
```

Comment:
Looks like this has already been true in other places, but it's best practice not to have styles in your html, but to have them over in the css/scss files.